src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -326,6 +329,7 @@ public void Push(Page page)
                 {
                     //Invoke Page events
                     page.InvokeAppeared();
+                    NotifyAccessibilityStatesChangeOfPages(null, page);
```

Comment:
I think this line needs to be changed as follows.
```
NotifyAccessibilityStatesChangeOfPages(curTop, page);
```

Reply:
Thank you for your comment.
The property VISIBLE of curTop is changed to FALSE at 315
Is the property VISIBLE of curTop FALSE at this point 332 ?
So it seems that this line needs to be moved to 315.

Reply:
I have checked that Disappeared called first and Appeared called next.
So you are right. Updated. Thank you for your review.